phpunit.xml.dist

```diff
@@ -1,19 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We should keep the dist file. If anyone want to change the settings they will copy phpunit.xml.dist to phpunit.xml. But the latter file should not be in git.